Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -5123,7 +5132,7 @@ AXRelationType AXObjectCache::attributeToRelationType(const QualifiedName& attri
 {
     if (attribute == aria_activedescendantAttr)
         return AXRelationType::ActiveDescendant;
-    if (attribute == aria_controlsAttr || attribute == popovertargetAttr)
+    if (attribute == aria_controlsAttr || attribute == popovertargetAttr || attribute == commandforAttr)
```

Comment:
Even here we like to keep things sorted so the order is not arbtrary:
```suggestion
   if (attribute == aria_controlsAttr || attribute == commandforAttr || attribute == popovertargetAttr)
```